hkmc2/shared/src/test/mlscript/ucs/patterns/Compilation.mls

```diff
@@ -0,0 +1,36 @@
+:js
+
+open annotations
+
+:e
```

Comment:
```suggestion
// * Note that `(@compile (...))` is currently parsed as `(@compile(...) ())`...
:e
```

Reply:
Manually fixed in 2dde44510ef5b335764f98c51622e676f9cc3088.